### DISCUSSION TITLE
25. April 2023

### DISCUSSION THREAD

**Discussion by mkow** [gramineproject/gramine#1292] [Meeting notes]
### Agenda

*(please write your proposed agenda items in comments under this discussion)*

- [ ] Mona: Review [Gramine roadmap project](https://github.com/orgs/gramineproject/projects/1/views/1) and update
- [ ] Mona: Discuss a general policy around adding dummy system calls
- [ ] Benny: Releases in `contrib` repo
- [ ] Dmitrii: new API in RA-TLS to extract additional data (in addition to `/dev/attestation/quote`)? Case in point: the JSON Web Token (JWT) sent back by Microsoft Azure Attestation, see https://github.com/gramineproject/contrib/pull/38
    - Proposal 1: simply use envvar like `RA_TLS_MAA_JWT` (no need for changes in core Gramine repo)
    - Proposal 2: extend `ra_tls_verify_callback_results` with a pointer to such blobs (needs changes in core Gramine repo)
    - Proposal 3: new RA-TLS API (function) to extract such blobs (needs changes in core Gramine repo)

**Comment by monavij** [comment]
1. Review Gramine roadmap project and update 
2. Discuss a general policy around adding dummy system calls
